src/features/create-quiz/create-quiz-second-card/create-quiz-second-card.tsx

```diff
@@ -0,0 +1,128 @@
+import { Button } from "primereact/button";
+import { InputText } from "primereact/inputtext";
+import { Card } from "primereact/card";
+import { Dropdown } from "primereact/dropdown";
+import { useState } from "react";
+import { AnswerItem } from "./components/answer-item";
+
+import { observer } from "mobx-react";
+import { QuizStore } from "../store/CreateQuizStore";
+
+export const CreateQuizSecondCard: React.FC = observer(() => {
+  const [value, setValue] = useState<string>("");
+
+  const [selectedCity1, setSelectedCity1] = useState(null);
```

Comment:
I suppose that this selectedCity state is for testing purposes only 😄 Just a reminder to not forget to adapt it for the actual data.